releasenotes/notes/fix-pillow-import-95bf52acd75fbb20.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Fixes a problem with importing the `pillow` module that
+    would make some of the drawers fail, including the
+    `dag` drawer` and the `latex` circuit drawer.
```

Comment:
You don't actually need a release note, the bug hasn't been released yet as it was introduced in #7525

Reply:
Got it. Done in https://github.com/Qiskit/qiskit-terra/pull/7774/commits/b89a04e2e03c156999ad94640bee13e6d50fa503.